packages/app/obojobo-repository/server/routes/dashboard.js

```diff
@@ -14,10 +14,20 @@ router
 	.route('/dashboard')
 	.get([requireCurrentUser, requireCanPreviewDrafts])
 	.get((req, res) => {
+		const cookies = req.headers.cookie.split(';')
+		let selectedSort = 'Alphabetical'
+
+		cookies.forEach(cookie => {
+			if (cookie.includes('selectedSort')) {
+				selectedSort = cookie.split('=')[1]
+			}
+		})
```

Comment:
Suggest using `array.find` since we're trying to pluck out a single value and don't need to continue looping after locating it.

```javascript
const cookieSort = cookies.find(cookie => {
  if(cookie.includes('selectedSort'){
    return cookie.split('=')[1]
  }
})
```

But before that, look at`cookies.get` https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/cookies/get  I think we can use it here instead.